user/admin.py

```diff
@@ -1,5 +1,28 @@
 from django.contrib import admin
+from django.contrib.admin import ModelAdmin
 from django.contrib.auth.admin import UserAdmin
 from .models import User
 
+
+class UserAdmin(ModelAdmin):
```

Comment:
The class name `UserAdmin` conflicts with the imported `UserAdmin` from `django.contrib.auth.admin`. Consider renaming the custom admin class to avoid confusion, such as `CustomUserAdmin`.